countdown.js

```diff
@@ -1,5 +1,15 @@
 function countdown(seconds){
   // ...
+  if (seconds >= 0) {
+    var i = window.setInterval(
```

Comment:
Why not just use a `setTimeout()` here, so that you don't need to stop it below?